client/store/playbackManager.ts

```diff
@@ -140,6 +141,32 @@ export const getters: GetterTree<PlaybackManagerState, RootState> = {
         return stream.Type === 'Subtitle';
       });
     }
+  },
+  getCurrentTime: (state) => {
+    return state.currentTime;
```

Comment:
No need for a getter that does no processing. Simply mapState for `currentTime`.